tasks/build-fits-site.yml

```diff
@@ -0,0 +1,83 @@
+- name: Build site for FITS microservice
+  blockinfile:
+    path: "/etc/apache2/conf-available/CrayFits.conf"
+    create: yes
+    block: |
+      Alias "/crayfits" "/var/www/html/CrayFits/public"
+      <Directory "/var/www/html/CrayFits/public">
+        FallbackResource /index.php
+        Require all granted
+        DirectoryIndex index.php
+        SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+      </Directory>
+
+- name: Enable FITS site
+  file:
+    src: "/etc/apache2/conf-available/CrayFits.conf"
+    dest: "/etc/apache2/conf-enabled/CrayFits.conf"
+    state: link
+
+- name: Change owner on html dir
+  file:
+    path: /var/www/html
+    owner: www-data
+    group: www-data
+
+- name: Download islandora FITS module
```

Comment:
Actually, I think this whole section can be removed. If I understand correctly this is installing the drupal-module. @dannylamb has done that [here](https://github.com/Islandora-Devops/claw-playbook/pull/129/files#diff-02360a2a7bb2306f59f8d5a35073947cR19) so this is unnecessary.